doc/rados/operations/crush-map-edits.rst

```diff
@@ -691,57 +700,55 @@ In order to ensure that the conversion is correct, there is a ``--compare`` comm
   maps appear equivalent
 
-If there were differences, the ratio of remapped inputs would be reported in
-the parentheses.
+If the command finds any differences, the ratio of remapped inputs is reported
+in the parentheses.
 
-When you are satisfied with the adjusted map, apply it to the cluster with a command of the form:
+When you are satisfied with the adjusted map, apply it to the cluster by
+running the following command:
 
 .. prompt:: bash $
 
    ceph osd setcrushmap -i adjusted
 
-Tuning CRUSH, the hard way
---------------------------
+Manually Tuning CRUSH
+---------------------
 
-If you can ensure that all clients are running recent code, you can
-adjust the tunables by extracting the CRUSH map, modifying the values,
-and reinjecting it into the cluster.
+If you have verified that all clients are running recent code, you can adjust
+the CRUSH tunables by extracting the CRUSH map, modifying the values, and
+reinjecting the map into the cluster. The procedure is carried out as follows:
 
-* Extract the latest CRUSH map:
+#. Extract the latest CRUSH map:
 
-  .. prompt:: bash $
+   .. prompt:: bash $
 
-	ceph osd getcrushmap -o /tmp/crush
+      ceph osd getcrushmap -o /tmp/crush
 
-* Adjust tunables.  These values appear to offer the best behavior
-  for both large and small clusters we tested with.  You will need to
-  additionally specify the ``--enable-unsafe-tunables`` argument to
-  ``crushtool`` for this to work.  Please use this option with
-  extreme care.:
+#. Adjust tunables. In our tests, the following values appear to result in the 
+   best behavior for both large and small clusters. The procedure requires that
+   you specify the ``--enable-unsafe-tunables`` flag in the ``crushtool``
+   command. Use this option with **extreme care**:
 
-  .. prompt:: bash $
+   .. prompt:: bash $
 
-     crushtool -i /tmp/crush --set-choose-local-tries 0 --set-choose-local-fallback-tries 0 --set-choose-total-tries 50 -o /tmp/crush.new
+      crushtool -i /tmp/crush --set-choose-local-tries 0 --set-choose-local-fallback-tries 0 --set-choose-total-tries 50 -o /tmp/crush.new
 
-* Reinject modified map:
+#. Reinject the modified map:
 
-  .. prompt:: bash $
+   .. prompt:: bash $
 
      ceph osd setcrushmap -i /tmp/crush.new
 
 Legacy values
 -------------
 
-For reference, the legacy values for the CRUSH tunables can be set
-with:
+To set the legacy values of the CRUSH tunables, run the following command:
 
 .. prompt:: bash $
 
    crushtool -i /tmp/crush --set-choose-local-tries 2 --set-choose-local-fallback-tries 5 --set-choose-total-tries 19 --set-chooseleaf-descend-once 0 --set-chooseleaf-vary-r 0 -o /tmp/crush.legacy
 
-Again, the special ``--enable-unsafe-tunables`` option is required.
-Further, as noted above, be careful running old versions of the
-``ceph-osd`` daemon after reverting to legacy values as the feature
-bit is not perfectly enforced.
+The special ``--enable-unsafe-tunables`` flag is required. Be careful when
```

Comment:
@jdurgin: This line says that the ``--enable-unsafe-tunables`` flag is required, but it's not included in the command that we've provided. What gives?